docs/api/fexpr.rst

```diff
@@ -190,6 +190,9 @@
         * - :meth:`.first()`
           - Same as :func:`dt.first()`.
 
+        * - :meth:`.isna()`
+          - Same as :func:`dt.math.isna()`.
```

Comment:
We were actually going to fully remove `dt.isna()` by keeping only `dt.math.isna()`: https://github.com/h2oai/datatable/issues/3037

Reply:
this is the method, not the function ---> `f.A.isna()`. Do you suggest I use `f.A.math_isna()`?

Reply:
We normally have `f.method()` to be equivalent to `dt.function()`. In the case when there is no `dt.function()`, i.e. no `dt.isna()`, I'm not sure we need to add the corresponding method.

However, now I'm starting to think as to why we actually wanted to deprecate `dt.isna()` and move it to `dt.math.isna()`. For instance, for string columns there is no math involved in checking if the value is NA or not...

Reply:
Ahhh nice observation... I'm not sure there is an issue raised for it. I think isna, should not be part of the math module, but directly under dt. 


Having methods is great and helps with readability/method chaining.

Reply:
@st-pasha Do you remember why we had to move `dt.isna()` to `dt.math`? To prevent having two separate functions for `isnan()` and `isna()`?

Reply:
Yeah, it's good for the moment. I will review the PR once it is green.

Reply:
The failure seems unrelated to this PR- it looks like a test on datetime is failing

Reply:
@samukweku sorry, I commented in a wrong thread. Anyways, the failure is related to the docs change

```
AssertionError: Variable doc_isna declared in documentation.h not found as a :cvar: among *.rst files
```
https://ci.appveyor.com/project/h2oops/datatable/builds/46911334/job/vg3lpfy24mcik9sx?fullLog=true